uri-templates/uri-templates.d.ts

```diff
@@ -1,18 +1,19 @@
-// Type definitions for uri-templates 0.1.2
+// Type definitions for uri-templates 0.1.5
 // Project: https://github.com/geraintluff/uri-templates
-// Definitions by: Bart van der Schoor <https://github.com/Bartvds>
+// Definitions by: Bart van der Schoor <https://github.com/Bartvds>, Juan Tamayo <https://github.com/jtamayo>
 // Definitions: https://github.com/borisyankov/DefinitelyTyped
 
-declare module 'uri-templates' {
-	function utpl(template: string): utpl.URITemplate;
+declare class UriTemplate {
```

Comment:
consider using an `interface` to define the ability to call a constructor with the `new` keyword.

also, consider using a common name-spacing pattern where all definitions are scoped within a `UriTemplates` namespace
```
declare module UriTemplates {
    interface UriTemplate {
        new(template: string): UriTemplate;
        (template: string): UriTemplate;
        // etc
    }
}

declare module "uri-templates" {
  export = UriTemplates.UriTemplate;
}
```